ui/ducks/send/send.js

```diff
@@ -1641,10 +1645,27 @@ const slice = createSlice({
       if (draftTransaction) {
         const isSwapAndSend = getIsDraftSwapAndSend(draftTransaction);
 
-        const { quotes } = draftTransaction;
+        const getIsIgnorableAmountError = () =>
+          [
+            INSUFFICIENT_TOKENS_ERROR,
+            INSUFFICIENT_FUNDS_ERROR,
+            INSUFFICIENT_FUNDS_FOR_GAS_ERROR,
+          ].includes(draftTransaction.amount.error) &&
+          !draftTransaction.sendAsset.balance;
+
+        const { quotes, gas } = draftTransaction;
         const bestQuote = quotes ? calculateBestQuote(quotes) : undefined;
+
+        const derivedGasPrice =
+          hexToDecimal(gas?.gasTotal || '0x0') > 0 &&
+          hexToDecimal(gas?.gasLimit || '0x0') > 0
+            ? new Numeric(gas.gasTotal, 16).divide(gas.gasLimit, 16).toString()
+            : undefined;
+
```

Comment:
Does the price need to be converted to gwei?

Seeing a bug where I have enough eth for gas, but the error is displayed
![Screenshot 2024-06-07 at 1 48 08 PM](https://github.com/MetaMask/metamask-extension/assets/100321200/31627987-a075-41ac-ae35-9cfbb35788c9)
![Screenshot 2024-06-07 at 1 48 16 PM](https://github.com/MetaMask/metamask-extension/assets/100321200/34254ff0-e24b-4553-b783-4578b3cb0fcc)